securedrop/tests/test_journalist.py

```diff
@@ -767,6 +767,27 @@ def test_user_resets_hotp(journalist_app, test_journo):
     assert old_secret != user.otp_secret
 
 
+def test_user_resets_user_hotp_format_odd(journalist_app, test_journo):
+    old_secret = test_journo['otp_secret']
+
+    with journalist_app.test_client() as app:
+        _login_user(app, test_journo['username'], test_journo['password'],
+                    test_journo['otp_secret'])
+
+        with InstrumentedApp(journalist_app) as ins:
+            app.post('/account/reset-2fa-hotp',
```

Comment:
Just a note for posterity that `uid=self.user.id` was removed in the dict here, but it was not actually needed as the view function in question uses the `g.user` object (this comment applies to the other uses of this `/account/reset-2fa-hotp` in this diff)